src/validate.ts

```diff
@@ -34,12 +35,15 @@ const getValidationResult = ({
 const inferRulesFromAttributes = (rules: any[], { inputs }: any) => {
     const extendedRules = [...rules];
 
-    forEach(inputs, (input: any) => {
-        if (get(input, 'validity.valid') === false) {
-            const rule = hasError(input.name, input.validationMessage);
-            extendedRules.push(rule);
+    for (const prop in inputs) {
+        if (inputs.hasOwnProperty(prop)) {
```

Comment:
I know eslint discourages this: https://eslint.org/docs/rules/no-prototype-builtins

Safe to use?

Reply:
good point, but as far as I understand we don't use `Object.create(null)` for inputs

Reply:
Cool, one to keep in mind I suppose if we get any issues.